manifest.json

```diff
@@ -5,18 +5,20 @@
     "author": "Sugar Labs",
     "description": "Learn to program music with snap-together blocks.",
     "icons": {
-	"512": "/activity/activity-icon-color-512.png",
-	"128": "/activity/activity-icon-color-128.png"
+        "512": "/activity/activity-icon-color-512.png",
+      	"128": "/activity/activity-icon-color-128.png"
     },
-    "app": {
-	"background": {
-	    "scripts": ["js/launch.js"]
-	}
+    "background": {
+       "scripts": ["js/background.js"]
     },
+    "web_accessible_resources": [
+        "index.html"
+    ],
     "permissions": [
-	"storage",
-	"unlimitedStorage",
-	"audioCapture",
```

Comment:
Was this necessary? We do have blocks that try to grab audio and video (although not critical at the moment to Music Blocks)

Reply:
Getting audio and video is unfortunately not a valid permission to give to an extension. However, the extension simply open the the html file packaged into the extension, and the block should still function simply by opening a dialog to request those permissions (if it works by opening `index.html`, it'll work in the extension)